packages/gatsby-theme-docs/globals.css

```diff
@@ -1,6 +1,8 @@
 @import '@commercetools-uikit/design-system/materials/custom-properties.css';
 
 :root {
-  --font-family-default: 'Roboto', sans-serif;
+  --font-family-default: 'Inter', sans-serif;
+  --font-feature-settings: 'ss02';
```

Comment:
Setting the feature setting globally to ensure that it's applied everywhere.

Reply:
This is not correct, sorry.   Disambiguation needs to be applied selectively to string literals and other places where it is important that the exact sequence of characters is seen.   In general body and UI  text you should use the font as designed by the font designer. 

Same for the line below,  you want to let the font do its letter spacing magic and only apply it where you want a bit wider spacing for disambiguation.  Except if @zbalek  really wants custom letter spacing everywhere including body text.